bundles/AdminBundle/Resources/public/js/pimcore/document/editables/image.js

```diff
@@ -242,7 +244,17 @@ pimcore.document.editables.image = Class.create(pimcore.document.editable, {
             } catch (e) {
                 console.log(e);
             }
-        }.bind(this));
+        }.bind(this),
+        function (res) {
+            const response = Ext.decode(res.response.responseText);
+            if (response && response.success === false) {
+                pimcore.helpers.showNotification(t("error"), response.message, "error",
+                    res.response.responseText);
+            } else {
+                pimcore.helpers.showNotification(t("error"), res, "error",
+                    res.response.responseText);
+            }
+        }.bind(this), [] ,this.type);
```

Comment:
```suggestion
        }.bind(this), [] ,this.getType());
```
we can kick the newly introduced property 😉

Reply:
Ah yes, I wanted to do it exactly like in the tag/image.js
Shouldn't we stick to one pattern? We could also use this property for the getType method?

Reply:
yes, I understand you tried to follow one pattern but then we have to introduce this property for abstract and other editables which requires more refactoring. we can just the existing method here with less code changes.